docs/_themes/clusterhq/layout_homepage.html

```diff
@@ -43,20 +43,20 @@
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
           </button>
-          <a class="navbar-brand" href="#"><img src="{{ pathto('_static/images/', 1) }}logo@2x.png" alt="ClusterHQ Container Data Management" /></a>
+          <a class="navbar-brand" href="https://clusterhq.com/"><img src="{{ pathto('_static/images/', 1) }}logo@2x.png" alt="ClusterHQ Container Data Management" /></a>
```

Comment:
All the other links to the home page are absolute. Note that I am changing this from a link that just takes you to the beginning of the page you are on (where you already are), into a link to the homepage.